src/utils/coordinates.js

```diff
@@ -96,7 +96,7 @@ module.exports.isCoordinate = function (value) {
 };
 
 function parseIfString (val) {
-  if (val.constructor === String) {
+  if (val != null && val.constructor === String) {
```

Comment:
If we're worried about nulls. This change should be all that's needed?

Also use `!==` not `!=`.

Reply:
I intentionally used '!=' because in this case both 'undefined' and 'null' will satisfy the condition. Do you think it is better to do 'if (val !== undefined && val !== null)' instead?

Reply:
Answering to the question "This change should be all that's needed?": you can get away with only this change, however, 'value.w' in parse function will get 'null' assigned (but value.w shouldn't be even defined, so this may cause potential side effects).

Reply:
Technically speaking, the change in this line is not mandatory, since the main checks are in 'parse' function and I've added it here  'to be safe'. We can remove this modification.

Reply:
better to use `!==` for consistency with the rest of the code base.